tgui/packages/tgui/interfaces/ClockworkSlab.js

```diff
@@ -0,0 +1,268 @@
+import { Fragment } from 'inferno';
+import { useBackend, useLocalState } from '../backend';
+import { Icon, Box, Button, Section, Table, Divider, Grid, ProgressBar, Collapsible } from '../components';
+import { Window } from '../layouts';
+import { TableRow } from '../components/Table';
+
+const convertPower = (power_in) => {
+  const units = ['W', 'kW', 'MW', 'GW'];
+  let power = 0;
+  let value = power_in;
+  while (value >= 1000 && power < units.length) {
+    power++;
+    value /= 1000;
+  }
+  return Math.round((value + Number.EPSILON) * 100) / 100 + units[power];
+};
+
+export const ClockworkSlab = (props, context) => {
+  const [selectedTab] = useLocalState(context, 'selectedTab', 'Servitude');
+  return (
+    <Window theme="clockwork" width={860} height={700}>
+      <Window.Content>
+        <Section
+          title={
+            <Box inline color={'good'}>
+              <Icon name={'cog'} rotation={0} spin={1} />
+              {' Clockwork Slab '}
+              <Icon name={'cog'} rotation={35} spin={1} />
+            </Box>
+          }>
+          <ClockworkButtonSelection />
+        </Section>
+        <div className="ClockSlab__left">
+          <Section height="100%" overflowY="scroll">
+            <ClockworkSpellList selectedTab={selectedTab} />
+          </Section>
+        </div>
+        <div className="ClockSlab__right">
+          <div className="ClockSlab__stats">
+            <Section height="100%" scrollable overflowY="scroll">
+              <ClockworkOverview />
+            </Section>
+          </div>
+          <div className="ClockSlab__current">
+            <Section
+              height="100%"
+              scrollable
+              overflowY="scroll"
+              title="Servants of the Cog vol.1">
+              <ClockworkHelp />
+            </Section>
+          </div>
+        </div>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const ClockworkHelp = (props, context) => {
+  return (
+    <Fragment>
+      <Collapsible title="Where To Start" color="average" open={1}>
+        <Section>
+          After a long and destructive war, Rat&#39;Var has been imprisoned
+          inside a dimension of suffering.
+          <br />
+          You are one of his last remaining, most loyal servants. <br />
+          You are very weak and have little power, with most of your scriptures
+          unable to function.
+          <br />
+          <b>
+            Install&nbsp;
+            <font color="#DFC69C">Integration Cogs&nbsp;</font>
+            to unlock more scriptures and siphon power!
+          </b>
+          <br />
+        </Section>
+      </Collapsible>
+      <Collapsible title="Unlocking Scriptures" color="average">
+        <Section>
+          Most scriptures require <b>cogs</b> to unlock.
+          <br />
+          Invoke&nbsp;
+          <font color="#DFC69C">
```

Comment:
That color string should be made into a const earlier in the file, so you can use the same one everywhere in that file without losing on maintainability and while improving readability.